### ISSUE TITLE
Cross thread Exception

### ISSUE THREAD

**Issue by ben1628** [RolandPheasant/DynamicData#235]
I have the following 

   private SourceList<PortfolioProp> _portfolioList { get; set; } = new SourceList<PortfolioProp>();
        private readonly IObservableCollection<PortfolioProp> _portfoilioCollection = new ObservableCollectionExtended<PortfolioProp>();
        public IObservableCollection<PortfolioProp> PortfoilioCollection => _portfoilioCollection;

In my function, I simply 
  PortfolioProp portfolioProp = new PortfolioProp()
            {
                Symbol = symbolName,
}
            _portfolioList.Add(portfolioProp);

            _portfolioList.Connect()
             .ObserveOn(RxApp.MainThreadScheduler)
            //.ObserveOnDispatcher()
            .Bind(_portfoilioCollection)
            .Subscribe();

I will get error:

Cross-thread operation not valid: Control accessed from a thread other than the thread it was created.

My understanding is MainThreadScheduler is the UI thread, how do I fix this problem?

**Comment by Claye-L** [comment]
Yu haven't mentioned what platform you are on, but i suggest you install the correct version  [the platform specific implementation of ReactiveUI] (https://github.com/reactiveui/ReactiveUI#nuget-packages) . I Have had similar problems because i had not installed ReaciveUI.WPF in a WPF App

**Comment by ben1628** [comment]
Oh, Thanks so much. It works.  You just saved me tons of time.

I'm using WinForm, When I started looking at using dynamicdata and ReactiveUI, I did add ReactiveUI into my winform Test application, and it was working. 

Forget to add this when I ported my application to use ReactiveUI and dynamic data.  I was so confused since my primary reason of using ReactiveUI is not having to deal with cross-thread.

Again thanks.
